apps/server/src/modules/provisioning/schulconnex-provisioning.config.ts

```diff
@@ -76,6 +78,9 @@ const config: SchulconnexProvisioningConfig = {
 	PROVISIONING_SCHULCONNEX_POLICIES_INFO_URL: Configuration.get('PROVISIONING_SCHULCONNEX_POLICIES_INFO_URL') as string,
 	RABBITMQ_URI: Configuration.get('RABBITMQ_URI') as string,
 	GEOGEBRA_BASE_URL: Configuration.get('GEOGEBRA_BASE_URL') as string,
+	SCHULCONNEX_COURSE_SYNC_HISTORY_EXPIRATION_SECONDS: Configuration.get(
```

Comment:
You also have to add this variable to the _schulconnex-group-provisioning-configmap.yml.j2_ and _schulconnex-group-removal-configmap.yml.j2_ so that the deployments can get the values from ansible and you also have to configure it in the _instance_config_all.yml_ in dof_app_deploy so that the value is defined and accessible to the deployments. Otherwise it will use the default value from the default.schema.json which is not changeable over the configmap.